### PR TITLE
Proposal for Issue Templates Overhaul

### PR DESCRIPTION
**Pull request by DJ-Katalina**
Let's try this again!

I am proposing an overhaul of all 5 current issue templates. I have worked on consistency, clarity, general grammar / spelling / typo corrections, etc.

Thank you. 👍

### CODE REVIEW

**Comment by DJ-Katalina** [comment]
Let's try this again!

I am proposing an overhaul of all 5 current issue templates. I have worked on consistency, clarity, general grammar / spelling / typo corrections, etc.

Thank you. 👍

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 1]
```diff
@@ -1,60 +1,57 @@
 ---
-name: Broken website
-about: Creates a broken website report about Dark Reader
-title: "[Broken Website] Replace with title"
+name: Broken Website Report
+about: Create a report about a website issue you experienced while using Dark Reader.
+title: "[Broken Website] Place title here"
```
Comment (Gusted) [inline .github/ISSUE_TEMPLATE/broken-website.md:4]: We actually had this on before-hand but changed to `Replace with title` as it was thought you would place the title after `place title here` So I would often see `[Broken Website] Place title here *Actually title*`

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 2]
```diff
@@ -1,60 +1,57 @@
...
+⚠⚠ Do not delete this issue template! ⚠⚠
+Reported issues must use this template and have all the necessary information provided. Incomplete reports are likely to be ignored and closed.
 -->
 
 <!--
-  Thank you for taking the time to report a broken website.
-  Please make sure there is no existing issue with this broken website.
-  Make sure to fill out every section when needed else remove the section.
-  Don't forget to add the website in the issue title.
+Thank you for taking the time to create a report about a website issue.
```
Comment (Gusted) [inline .github/ISSUE_TEMPLATE/broken-website.md:15]: Let's preserve the whitespace.

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 3]
```diff
@@ -1,60 +1,57 @@
...
-<!-- The link to the website where you can observe the issue. -->
+# Broken Website Report
 
+## Website Address
+<!-- Provide the website address to allow us to troubleshoot this website issue. -->
 
-## How to Reproduce
-<!-- We need to know how you encountered the bug to properly troubleshoot the issue. -->
+## Steps To Reproduce
+<!-- Provide steps to reproduce to allow us to troubleshoot this website issue. -->
```
Comment (Gusted) [inline .github/ISSUE_TEMPLATE/broken-website.md:28]:
```suggestion
<!-- Provide steps to reproduce the problem, and allowing us to troubleshoot this website issue. -->
```

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 4]
```diff
@@ -1,60 +1,57 @@
...
-## Actual behavior
-<!-- A clear and concise description of what actually happened. -->
-
+## Actual Behavior
+<!-- Provide a clear and concise description of what happened. -->
 
 ## Screenshots
-<!-- If applicable, add screenshots to help explain your problem. -->
-
+<!-- Add screenshots to help explain this website issue. -->
```
Comment (Gusted) [inline .github/ISSUE_TEMPLATE/broken-website.md:43]: I think we should preserve the `If applicable,` as not every issue would need it and I don't want screenshot when not needed.

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 5]
```diff
@@ -1,60 +1,57 @@
...
 
 ## System Information
 <!--
-  Please add a version of the browser you are using. 
-  If you don't know how to get your browser/Dark Reader version please search it up online.
+Specify the browser name and version as well as the Dark Reader version you are
```
Comment (Gusted) [inline .github/ISSUE_TEMPLATE/broken-website.md:47]: Whitespace ;)

**Review by Gusted — CHANGES_REQUESTED** [review]
Reviewed 1 template but most comments can be applied to the other ones.

**Comment by DJ-Katalina** [comment]
Let me fix a few more issues before review. Sorry.

**Comment by DJ-Katalina** [comment]
Ok, I believe I am ready for review.

**Review by Gusted — APPROVED** [review]

**Pull request merged**